homework/index.js

```diff
@@ -29,23 +31,127 @@
     return elem;
   }
 
+  function addRow(table, title, repoText) {
+    const tr = createAndAppend('tr', table);
+    createAndAppend('th', tr, {
+      text: title,
+    });
+    createAndAppend('td', tr, { text: repoText });
+    return tr;
+  }
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const date = new Date(repo.updated_at);
+    const li = createAndAppend('li', ul, { class: 'li-root' });
+    const table = createAndAppend('table', li);
+    const firstRow = addRow(table, 'Repository:', '');
+    createAndAppend('a', firstRow.lastChild, {
+      href: repo.html_url,
+      text: repo.name,
+      target: '_blank',
+    });
+
+    addRow(table, 'Description:', repo.description, 'td');
+    addRow(table, 'Forks:', repo.forks, 'td');
+    addRow(table, 'Updated:', date.toLocaleDateString(), 'td');
+  }
+
+  function createOptions(repo, parent, index) {
+    createAndAppend('option', parent, {
+      text: repo.name,
+      value: index,
+    });
+  }
+  function selections(selection, data, parent) {
```

Comment:
1. This function name is rather vague. What does this function do? That is what the name should tell us.
2. The name `selection` (first param) is also rather vague. What does it contain? I think the name `selectedIndex` would make it clear what it contains.
3. Don't use the generic name `data` (second param) if you know that this function actually receives an array of repository objects. Therefore,  the name `repositories` would seem a good choice.